libraries/passive_sensitive.rb

```diff
@@ -0,0 +1,13 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: passive_sensitive.rb
+
+require 'chef/resource'
+
+class Chef
+  class Resource
+    def sensitive(arg)
```

Comment:
The thought is that the [real definition](https://github.com/chef/chef/blob/12.0.3/lib/chef/resource.rb#L317) is `def sensitive(arg=nil)` (to satisfy rubocop we could do `def sensitive(_ = nil)` ) Which enables invoking both with and without a single argument. `def sensitive(*)` would enable passing more arguments than the real definition.